packages/objects-horizontal-list/_horizontal-list.scss

```diff
@@ -63,7 +95,11 @@ $horizontal-list-delimiter: ',\00A0' !default;
 ///
 .#{$scales-namespace}#{$scales-objects-class-namespace}HorizontalList--delimited {
   > li {
+    margin-right: 0;
+
     &::after {
+      margin-left: $horizontal-list-delimiter-spacing / 2;
```

Comment:
I'm not sure this is going to work. I think it makes sense when the `|` delimiter is used, but you can change it to other characters. If it is a `,` it would be weird to have half of the space on either side...you'd want it all on the right. Any ideas?